UnitTesting/CalcStats.cs

```diff
@@ -0,0 +1,95 @@
+﻿
+namespace UnitTesting
+{
+    public static class CalcStats
```

Comment:
Make class regular, not static and add interface. The main idea of Task is TDD approach. So, Firstly you should create Interface, than write tests, and after write implementation. For other files as well.